.github/workflows/test.yml

```diff
@@ -21,7 +21,7 @@ jobs:
 
       - run: pip install poetry codecov
       - name: Install dependencies
-        run: poetry install --extras "django flask opentelemetry"
+        run: poetry install --extras "debug-server django flask pydantic opentelemetry"
```

Comment:
I think by default poetry installs dev dependencies, shall add the debug-server dependencies as dev deps too? so we don't have to specify it here? :)

Reply:
oh, that's nice! I'm coming from pipenv where you have to be explicit to ensure dev deps are installed. I'll trial the change and then send an update if it works out.

Typically should I rebase the commit into the right place or do you do squash merges when it is time?

Reply:
@ossareh awesome! no strong preference on rebasing vs squashing, but I think we leaning towards squashing the PRs /cc @BryceBeagle :)